pkg/oc/clusterup/run_self_hosted.go

```diff
@@ -214,15 +213,6 @@ func (c *ClusterUpConfig) StartSelfHosted(out io.Writer) error {
 		return err
 	}
 
-	// bootstrap the service-ca operator with the legacy service-signer.crt and key.
-	err = createServiceCASigningSecret(clientConfig,
```

Comment:
@deads2k what create these?

Reply:
> @deads2k what create these?

They are no longer needed since we no longer proxy the webconsole.